include/CCConst.h

```diff
@@ -34,8 +33,16 @@ namespace CCCoreLib
 	constexpr double DEG_TO_RAD = (M_PI/180.0);
 	
 	//! Numerical threshold for considering a value as "zero"
-	[[deprecated( "Use std::numeric_limits<>::epsilon or CCCoreLib::LessThanEpsilon()/CCCoreLib::GreaterThanEpsilon()" )]]
-	constexpr double ZERO_TOLERANCE = static_cast<double>(FLT_EPSILON);
+	[[deprecated( "Use ZERO_TOLERANCE_F/ZERO_TOLERANCE_D or CCCoreLib::LessThanEpsilon()/CCCoreLib::GreaterThanEpsilon()" )]]
+	constexpr double ZERO_TOLERANCE = std::numeric_limits<float>::epsilon();
+	
+	//! ZERO_TOLERANCE_F is used to set a float variable to "close to zero".
+	constexpr float ZERO_TOLERANCE_F = std::numeric_limits<float>::epsilon();
+	
+	//! ZERO_TOLERANCE_D is used to set a double variable to "close to zero".
+	//! It is defined as std::numeric_limits::epsilon() because using
```

Comment:
Is github also hiding the '<'float'>' or '<'double'>' in the comments?!

Reply:
lol - no - I copy-pasted the comment from above :-)